scripts/background.js

```diff
@@ -273,40 +274,33 @@ chrome.runtime.onMessage.addListener(function(message,sender,sendResponse){
                 chrome.tabs.sendMessage(tabs[0].id, {RTurl:RTurl});
                 console.log(RTurl);
             });
-        }else if(message.message=='changeBadge'){
+        }else if(message.message=='changeBadge'){ //Used to change bage for auto-archive feature
           var tabId=message.tabId;
-          console.log(tabId);
           chrome.browserAction.setBadgeText({tabId: tabId, text:"\u2713"});
           console.log("Badge Changed");
         }else if(message.message=='showall'){
           chrome.storage.sync.get(['show_context'],function(event){
             if(!event.show_context){
               event.show_context="tab"; //By-default the context-window open in tabs
             }
-            chrome.storage.sync.get(['annotation_context'],function(event){
-              if(event.annotation_context==undefined){
-                event.annotation_context="domain"; 
-              }
-            });
             var received_url=message.url; //URL which is received by message-parsing_url
             received_url = received_url.replace(/^https?:\/\//,'');
-            var length=received_url.length; 
             var last_index=received_url.indexOf('/');
             var url=received_url.slice(0,last_index);    //URL which will be using for alexa and whois
             var open_url=received_url;          //URL which will be needed for finding tweets
             if(open_url.slice(-1)=='/') open_url=received_url.substring(0,open_url.length-1); 
             chrome.storage.sync.get(['auto_update_context'],function(event1){
               if(event1.auto_update_context==undefined){
-                event1.auto_update_context=false;
+                event1.auto_update_context=false; //By default auto-update context is off
               }
               console.log("here");
```

Comment:
Do we need to keep this in the code? You used it while developing, its best to avoid this kind of generic msgs in the production code.